keyboards/binepad/candypad/config.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 binepad (@binepad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* ----- Encoder ----- */
+#define ENCODER_DEFAULT_POS 0x3 // enable 1:1 resolution
+
+/* ----- OLED ----- */
+#ifdef OLED_ENABLE
+/* I2C (for OLED) */
+#    define I2C1_SCL_PIN GP23
+#    define I2C1_SDA_PIN GP22
+#    define I2C_DRIVER I2CD1
+
+/* Configure oled driver for the 128x32 oled */
+#    define OLED_TIMEOUT (15 * 60 * 1000) // 15 minutes
+#    define OLED_UPDATE_INTERVAL 33       // ~30fps
+
+/* Font */
+#    define OLED_FONT_H "keyboards/binepad/candypad/glcdfont.c"
+#    define OLED_FONT_END 127
```

Comment:
if only using the basic ascii section of the font file, why bother including a font file?  Just use the default font.

Reply:
The fonts have differences, e.g.
OG -vs.- This One
![image](https://github.com/user-attachments/assets/a18a2e65-7748-4dab-971a-11b0b37777ac) 
![image](https://github.com/user-attachments/assets/28b5178e-8f3c-4fd4-a97f-68bfd2d3863f)

...but I suppose I can move that to the vendor userspace.